src/test/java/com/example/bookstore/repository/BookRepositoryTest.java

```diff
@@ -0,0 +1,100 @@
+package com.example.bookstore.repository;
+
+import com.example.bookstore.model.Book;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
```

Comment:
I have data already added through the libbase scripts, we don't change them during development, we just add other scripts, so I decided to test them right away since it takes less time to write tests and in my opinion is quite valid